### DISCUSSION TITLE
Internationalization - help needed

### DISCUSSION THREAD

**Discussion by Christopher-Hayes** [Christopher-Hayes/vscode-chatgpt-reborn#20] [Announcements]
## i18n support

I've added initial i18n support in this extension `v3.15.0`. For a limited number of languages, I've added some basic Google Translate translations. But, I know they are going to be far from perfect.

### Help needed

So, if you are able to help at all with translating to a language, I would happily appreciate the contribution.

### The ways to do this:
1. "Fork" this repo into your account. Update the .json for the language you are updating in the "/supported-languages" folder. Then in this repo, make a "pull request" with your repo -> this repo.
2. If you're not comfortable with PRs and editing files. Just add a comment below with the translations that need to be fixed, and I'll update the .json files.

**Comment by zzy-life** [comment]
![image](https://user-images.githubusercontent.com/68179477/235427573-08cc3109-0698-4332-971e-273bab7b99f1.png)
Right click not internationalized

**Reply by Christopher-Hayes** [reply]
Good call out, thank you. Will try to fix for next release.
